### PR TITLE
ci: moves ActiveMQ and RabbitMQ to new docker images

### LINKED ISSUE #3633: flakey test: ITActiveMQSender
ITActiveMQSender loops trying to connect to the VM transport in CI sometimes.

Here's an example from the JDK11 job
```
2023-12-13 04:59:18:131 [ActiveMQ Task-2] WARN org.apache.activemq.transport.failover.FailoverTransport - Failed to connect to [vm://embedded-broker?create=false] after: 30 attempt(s) with Broker named 'embedded-broker' does not exist., continuing to retry.
```

We should probably switch to a docker test image approach as we already have problems with this test library as it is pinned to junit4.

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Notably, this removes a JUnit 4 dependency in activemq and should fix flakes related to ActiveMQ integration tests.

Fixes #3633

After this, I will rebase #3635 and hopefully it doesn't flake anymore. If so, I'll merge that and cut the pending minor.

### CODE REVIEW

File: zipkin-collector/activemq/src/test/java/zipkin2/collector/activemq/ITActiveMQCollector.java [hunk 1]
```diff
@@ -183,12 +192,12 @@ void messageWithMultipleSpans(SpanBytesEncoder encoder) throws Exception {
       }
     };
 
-    activemq.pushMessage(collector.queue, PROTO3.encodeList(spans));
-    activemq.pushMessage(collector.queue, PROTO3.encodeList(spans)); // tossed on error
-    activemq.pushMessage(collector.queue, PROTO3.encodeList(spans));
-
     collector = builder().storage(buildStorage(consumer)).build().start();
 
+    pushMessage(collector.queue, PROTO3.encodeList(spans));
```
Comment (codefromthecrypt) [inline zipkin-collector/activemq/src/test/java/zipkin2/collector/activemq/ITActiveMQCollector.java:197]: moved this as I needed a non stopped collector instance. It doesn't change the nature of the test

**Comment by codefromthecrypt** [comment]
oops forgot to click to make them public
https://github.com/orgs/openzipkin/packages/container/zipkin-activemq/settings
https://github.com/orgs/openzipkin/packages/container/zipkin-rabbitmq/settings

**Comment by codefromthecrypt** [comment]
rebased

**Comment by codefromthecrypt** [comment]
kicked the JDK 11 build. I'm not sure what it hung on, but it spent 10 minutes with no feedback in the zipkin-server module. This isn't a repeating flake pattern so far..

**Review by anuraaga — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
thanks for the support for the last week plus @anuraaga!

**Pull request merged**